src/components/pages/RoleSignup/Applications/Mentee.js

```diff
@@ -70,394 +71,413 @@ const Mentee = ({ dispatch, error, successPage }) => {
+                            {country}{' '}
                           </Option>
                         ))}
                       </Select>
                     </Form.Item>
-                  </div>
+                  </Col>
+                </Row>
 
-                  <Form.Item
-                    label="City"
-                    type="text"
-                    name="city"
-                    rules={[
-                      {
-                        required: true,
-                        message: 'City is required!',
-                      },
-                    ]}
-                    value={formValues.city}
-                    onChange={handleChange}
-                    style={{ margin: '0 1rem .5rem 0' }}
-                  >
-                    <Input placeholder="Your city" />
-                  </Form.Item>
-                </Col>
-              </Row>
+                <Row>
+                  <Col md={15} xs={24} offset={1}>
+                    <div className="locationUS">
+                      <Form.Item
+                        label="State"
+                        className="form-group"
+                        name="state"
+                        labelCol={{ span: 10 }}
+                        wrapperCol={{ span: 10 }}
+                        style={{ margin: '0 1rem .5rem 0', gap: '0rem' }}
+                        rules={[
+                          {
+                            required: true,
+                            message: 'State is required!',
+                          },
+                        ]}
+                      >
+                        <Select
+                          showSearch
+                          placeholder="- Select -"
+                          onChange={e => handleChange(e, 'select', 'state')}
+                        >
+                          {states.map(state => (
+                            <Option key={state} value={state}>
+                              {' '}
+                              {state}{' '}
+                            </Option>
+                          ))}
+                        </Select>
+                      </Form.Item>
+                    </div>
 
-              <hr />
+                    <Form.Item
+                      label="City"
+                      type="text"
+                      name="city"
+                      className="form-group"
+                      labelCol={{ span: 10 }}
+                      rules={[
+                        {
+                          required: true,
+                          message: 'City is required!',
+                        },
+                      ]}
+                      value={formValues.city}
+                      onChange={handleChange}
+                      style={{ margin: '0 1rem .5rem 0' }}
+                    >
+                      <Input placeholder="Your city" />
+                    </Form.Item>
+                  </Col>
+                </Row>
```

Comment:
In this section, wouldn't it be more intuitive for the user to enter city first, then state, then country? That would be the expected order if someone is used to U.S. address formatting.